src/routes/checkout/Address/Address.js

```diff
@@ -1,5 +1,317 @@
-import React from "react";
+import React, { useState, useContext, useEffect, useRef } from "react";
 
+import { onAuthStateChanged } from "firebase/auth";
+import { AppContext } from "../../../context/app-context";
+import { UserContext } from "../../../context/user/user-context";
+import { SET_CHECKOUT_ADDRESS } from "../../../constant";
+
+import { doc, getDoc, setDoc } from "firebase/firestore";
+
+import {
+  ThemeProvider,
+  Container,
+  Typography,
+  Grid,
+  Autocomplete,
+  TextField,
+  Button,
+  CircularProgress,
+} from "@mui/material";
+import { ButtonStyles, responsiveTheme, textFieldStyles } from "./muiStyles";
 export default function Address() {
-  return <div>Address</div>;
+  const { dispatch: userDispatch, state: userState } = useContext(UserContext);
+  const { auth, db } = useContext(AppContext);
+
+  const [loading, setLoading] = useState(false);
+
+  // Reference values of inputs
+  const buildingRef = useRef();
+  const apartmentNumberRef = useRef();
+  const messageRef = useRef();
+
+  console.log(userState);
+
+  // Set shipping_address data from database in user context
+  useEffect(() => {
+    const userData = userState;
```

Comment:
You don't need this part, like, the entire loading shipping address part here. All user's data from the database have been initialized to Context in the `user-handler.js`, which runs when the page first loaded up